packages/test/test-end-to-end-tests/src/test/idCompressor.spec.ts

```diff
@@ -710,7 +710,7 @@ describeCompat("IdCompressor in detached container", "NoCompat", (getTestObjectP
 describeCompat("IdCompressor Summaries", "NoCompat", (getTestObjectProvider) => {
```

Comment:
I don't know if there's another file doing this, but I would love to just see a side-by-side comparison that an id (datastore, docId, containerId) is shorter than the normal `uuid` we generate in an e2e test. For example, I don't exactly know how it works, but after id's have been finalized, it would be nice to see that `datastore.id.length < uuid().length` or something along those lines.

Reply:
Good call out - will add!